### PR TITLE
balances bluespace miners

### PR DESCRIPTION
**Pull request by bloons3**
## About The Pull Request

The current meta is for science to get the AI to give them access to the teleporter, where an enterprising Scientist will destroy the machine in order to get at the bluespace crystals within. Then Bluespace miners will be tech rushed, and bluespace miners will soon fill every square meter of the station.

image

This PR makes Bluespace miners a late game item by tweaking their cost, and moves them to being a cargo only item, since neither Science nor Engineering are in the business of harvesting materials for the station.

This should also discourage the current meta of players deconstructing the teleporters roundstart for their bluespace crystals in order to build a bluespace miner, which is detrimental to several other mechanics (like actually using the teleporter network).

Screenshot taken from a High-Population MRP round: 
image

## Why It's Good For The Game

machines that generate infinite resources shouldn't be possible to create at the start of the round by stealing bluespace crystals from the teleporter

It is not the job of science or engineering to get materials for the station

End game machines should only be possible mid to late round

## Changelog
:cl:
balance: rebalanced bluespace miner
/:cl:

### CODE REVIEW

**Comment by bloons3** [comment]
I made it 11 because there are actually 2 sets of teleporters on the station and you can get 10 total crystals from destroying them

**Comment by ike709** [comment]
Better solution is to just remove bluespace crystal harvesting entirely. Have them break when the machine is deconstructed.

**Comment by eeSPee** [comment]
But... Why?

**Comment by bloons3** [comment]
> Better solution is to just remove bluespace crystal harvesting entirely. Have them break when the machine is deconstructed.

Ruko is bringing up that the bad part is that these machines self-replicate, since they create diamonds which can be converted into BS crystals. What if you couldn't print the board, but only order it as mystery tech DLC from CentCom?

**Comment by ghost** [comment]
just destroy the BS in the roundstart teleporters

**Comment by eeSPee** [comment]
IMO a more effective solution would be:
* Reduce the harvest rate of BS miners
* Have them require 5 BS crystals
* Limit what mats they can mine depending on the current tier parts: T1 iron and glass,  T2 copper and silver, T3 plasma and gold, T4 diamond and unranium 

This way they are still useful, but can no longer be used to replace miners, until lategame till you get the tech and mats.

**Comment by Effemu** [comment]
please tell me how the fuck is mining supposed to be enjoyable if all you do is mine rocks for majority of the round

**Comment by metroid1310** [comment]
As a certified lowpop player, I'd like to say that not having bluespace miners is a nightmare if anyone (like science, the people who make machines actually function) wants anything that needs mats, but there's no shaft miners, command staff, or cargonians. 
This is seriously gonna affect the enjoyability of lowpop because it's fairly uncommon a round will have someone who plays lowpop shaft miner and nobody else wants to buck up and get mats, or can't without tiding the entire way there and back. 
BS miners are the only thing that keep me sane because they're the only realistic way for the station to get anything but iron, glass, or copper in quite a few rounds

**Comment by jlsnow301** [comment]
These were basically created (thanks Ike) to address the problem with mining being non existent (low pop) or dead (it's a newbie server)

A single working miner makes these obsolete. Removing them or delaying them in any aspect will simply illustrate how bad of a design mining is to the game.

Bsminers are not the problem, they're a bandaid

**Comment by ghost** [comment]
> please tell me how the fuck is mining supposed to be enjoyable if all you do is mine rocks for majority of the round

NSV Miners miner without the incentive of gamer loot.

**Comment by Effemu** [comment]
> 
> 
> > please tell me how the fuck is mining supposed to be enjoyable if all you do is mine rocks for majority of the round
> 
> NSV Miners miner without the incentive of gamer loot.

obraz
and how is that working out for them

**Comment by francinum** [comment]
> > > please tell me how the fuck is mining supposed to be enjoyable if all you do is mine rocks for majority of the round
> > 
> > 
> > NSV Miners miner without the incentive of gamer loot.
> 
> obraz
> and how is that working out for them

Hi, NSV host here.

That's not remotely the reason we're offline right now.

Please get actual arguments.

**Comment by pixelman77** [comment]
Will probably make low pop a nightmare to play, miners usually die quick or refuse to do their job and instead rush to kill megafauna, and that's only when someone actually picks miner.

**Comment by Fronsis** [comment]
I guess this is another thing to add on the checklist of PR's that give good stuff for MRP but bad for LRP

**Comment by Effemu** [comment]
> 
> 
> I guess this is another thing to add on the checklist of PR's that give good stuff for MRP but bad for LRP

I wouldnt call it good for mrp, being a miner basically takes you out of the round, so you cant really roleplay. At least now you can come back to station after sci makes enough bsms to make station self-sufficient

**Comment by francinum** [comment]
I am still highly against allowing them to exist in any capacity.

If mining doesn't want to do their job, then mining clearly has a problem and needs to be adjusted.

How many times do I have to say this:
*If a system is bad, adding a shortcut around it is never the solution.*

**Comment by bloons3** [comment]
> sci makes enough bsms to make station self-sufficient

This, is bad.

**Comment by Moccha-Bee** [comment]
The main reason people overlook why there is barely any ores is first is that bee is a new-player focused server and when they go to lavaland they get introduced to various threats that can easily kill them if they don't know what they are doing, so more often than not miners simply get killed not because they were going for megafauna loot or anything but just because they got stunned by a goliath tentacle.

**Review by utilbird — APPROVED** [review]

**Comment by Cartlord** [comment]
Pasting this here because I accidentally posted it earlier on a different BS miner nerf.
TL;DR Bluespace Miners are something I think Golden should be able to have at roundstart, because it's a newbie/n00b server and those aren't the people who play Shaft Miner.
Almost everyone I've asked about it on Golden hates this change, yet none have commented here, so I figured I'd post grievances both from them and from myself here.
I will open this by admitting I usually play on lowpop Golden, late at night - pop varies, when I'm active, between ~22 and ~4. In very lowpop scenarios, I find Bluespace Miners are almost necessary for things such as upgrading machinery. I think it's ridiculous that they require mining materials to create, given that (as far as I've seen) their main use is in rounds where you don't HAVE Shaft Miners, and you need everyone on-board just to keep the station running safely. I think Bluespace Miners are necessary on Golden lowpop, and nerfing them so hard just neuters the possibilities.
Also consider: from what I've gathered, Golden is meant to be, or seen as, good for new players; new players who shouldn't be playing Shaft Miner, and therefore could use the free mats from a Bluespace Miner.
The main counterpoints I've heard to buffing Bluespace Miners are either something about highpop, or something about them being able to effectively self-replicate; in my opinion, the latter can be solved both A) by limiting the number of machine boards present in a round (already implemented on Golden) and B) by making them unable to produce bluespace crystals; the former, however, is something I see as a COMPLETE non-issue. I've never seen Golden highpop, it's (in my personal, admittely-limited experience) a medium to lowpop server for new players.
I would like Bluespace Miners to be something we can make at least one of at roundstart, so that we can reliably get materials for things like machine parts without needing one of the ten people online to become a ghetto Shaft Miner - which often would require someone changing job or access, which itself would require breaking into Cap to get the spare AA since there isn't often Cap or HoP online.
